orchagent/switchorch.cpp

```diff
@@ -12,7 +13,9 @@ const map<string, sai_switch_attr_t> switch_attribute_map =
 {
     {"fdb_unicast_miss_packet_action",      SAI_SWITCH_ATTR_FDB_UNICAST_MISS_PACKET_ACTION},
     {"fdb_broadcast_miss_packet_action",    SAI_SWITCH_ATTR_FDB_BROADCAST_MISS_PACKET_ACTION},
-    {"fdb_multicast_miss_packet_action",    SAI_SWITCH_ATTR_FDB_MULTICAST_MISS_PACKET_ACTION}
+    {"fdb_multicast_miss_packet_action",    SAI_SWITCH_ATTR_FDB_MULTICAST_MISS_PACKET_ACTION},
+    {"ecmp_default_hash_seed",              SAI_SWITCH_ATTR_ECMP_DEFAULT_HASH_SEED},
```

Comment:
-> ecmp_hash_seed

Reply:
agree. default shall not be in the attribute name. default is a value not an attribute. the SAI attribute name is not perfect.